aFWall/src/main/java/dev/ukanth/ufirewall/util/LogNetUtil.java

```diff
@@ -75,24 +81,34 @@ protected String doInBackground(NetParam... params)
 
                         try{
 
-                            /* [WIP] looking for ping timeout option */
-                            String result = output(Runtime.getRuntime().exec(String.format(PING_CMD, /*G.logPingTimeout(), */params[0].address)));
+                            // using libsuperuser
+                            // This will need permission in AFWall+
+                            // "0:(root) Apps running as root"
+
+                            String command = String.format(PING_CMD, Api.getBusyBoxPath(context, true), G.logPingTimeout(), params[0].address);
+
+                            Log.d(TAG, "Execute CMD: " + command);
 
-                            if(result.isEmpty()){
+                            String shell_result = parse(Shell.run("su", new String[]{
```

Comment:
Ok, I've change "su" to "sh" to be like this:
`String shell_result = parse(Shell.run("sh", new String[]{ ...`
But, ping returns an error:
![tag alt](http://vzool.net/img/device-2017-01-28-094409.png)